resources/website-l10n.xsl

```diff
@@ -85,21 +85,21 @@
         <l:template name="section" text="%t"/>
       </l:context>
       <l:context name="edit-me">
-        <l:template name="edit-me-title" text="Edit this page on GitHub" />
-        <l:template name="edit-me-text" text="edit" />
+        <l:template name="edit-me-title" text="在 GitHub 上编辑本页" />
+        <l:template name="edit-me-text" text="编辑" />
       </l:context>
       <l:context name="meta">
-        <l:template name="meta-description" text="Get started with the documentation for Elasticsearch, Kibana, Logstash, Beats, X-Pack, Elastic Cloud, Elasticsearch for Apache Hadoop, and our language clients." />
+        <l:template name="meta-description" text="有关如何使用 Elasticsearch、Kibana、Logstash、Beats、X-Pack、Elastic Cloud、Elasticsearch for Apache Hadoop 及我们各种语言的客户端的文档。" />
       </l:context>
       <l:context name="annotation">
-        <l:template name="added" text="Added in " />
+        <l:template name="added" text="于 $version 添加。$custom_text" />
```

Comment:
Sorry @medcl, I didn't make this clear.

With docbook, i can't substitute parameters into a translation, eg I can have a translation like "Added in %version.  %custom_text".  So what I have to do is to concatenate separate pieces, eg:

* translate(added-in)
* $version
* translate(sentence-separator)
* $custom_text

So you need to translate `added-in` as the chinese equivalent of "Added in".  This will have a version number appended to it.  And then the `sentence-separator`, then some custom text, if there is any.

You don't have the equivalent of a full stop `.` in Chinese, do you?  If so, the sentence separator can be blank.

Does this make sense?